### PR TITLE
Added missing test for ButtonLayout className prop

### PR DESCRIPTION
**Pull request by rghorbani**
### What changed

Added missing test requested by @hepiyellow in #2283. Missing in #2203.

### Why it changed

...

---

- [x] Change is tested
- [x] Change is documented
- [x] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

File: src/ButtonLayout/ButtonLayout.spec.js [hunk 1]
```diff
@@ -35,6 +35,19 @@ describe('ButtonLayout', () => {
...
+  it('should have custom className', () => {
+    const driver = createDriver(
+      <ButtonLayout className="myClass">
+        <div>
+          abc
+        </div>
+      </ButtonLayout>
+    );
+
+    expect(driver.exists()).toEqual(true);
```
Comment (hepiyellow) [inline src/ButtonLayout/ButtonLayout.spec.js]: Testing `exists()` is redundant

File: src/ButtonLayout/ButtonLayout.spec.js [hunk 2]
```diff
@@ -35,6 +35,19 @@ describe('ButtonLayout', () => {
...
+    expect(driver.doesComponentHasClass('myClass')).toEqual(true);
```
Comment (hepiyellow) [inline src/ButtonLayout/ButtonLayout.spec.js]: Use `toBeTruthy()`

File: src/ButtonLayout/ButtonLayout.spec.js [hunk 3]
```diff
@@ -35,6 +35,19 @@ describe('ButtonLayout', () => {
     expect(driver.getComponentAttribute('href')).toEqual(href);
   });
 
+  it('should have custom className', () => {
+    const driver = createDriver(
+      <ButtonLayout className="myClass">
+        <div>
```
Comment (hepiyellow) [inline src/ButtonLayout/ButtonLayout.spec.js]: Is this `<div/>` necessary for this test?

File: src/ButtonLayout/ButtonLayout.spec.js (hunk 2 above)
Comment (rghorbani) [reply src/ButtonLayout/ButtonLayout.spec.js]: done

File: src/ButtonLayout/ButtonLayout.spec.js (hunk 1 above)
Comment (rghorbani) [reply src/ButtonLayout/ButtonLayout.spec.js]: removed

File: src/ButtonLayout/ButtonLayout.spec.js (hunk 3 above)
Comment (rghorbani) [reply src/ButtonLayout/ButtonLayout.spec.js]: It produces an error without any children.

Comment (rghorbani) [reply src/ButtonLayout/ButtonLayout.spec.js]: because of usage of 
`children.props.className` and `children.props.style` in `ButtonLayout.js`

**Pull request merged**